sale_configurator_base/templates/sale_report_templates.xml

```diff
@@ -105,8 +100,18 @@
         <xpath expr="//td[@name='td_subtotal']" position="replace">
             <td name="td_subtotal" class="text-right o_price_total">
                 <t
-                    t-if="not line.child_ids and not line.report_empty_parent and line.parent_id"
-                >
+                    t-set="is_child"
+                    t-value="line.parent_id and (not line.child_ids) and (not line.report_line_is_empty_parent)"
```

Comment:
a line with a parent can not have child so testing line.parent_id is enough (same for report_line_is_empty_parent this have no meaning for child line